misc/stackblitz-gen.js

```diff
@@ -57,15 +58,16 @@ function generateIndexHtml() {
   return `<!DOCTYPE html>
 <html>
 
-  <head>
+  <head>xx
```

Comment:
xx?

Reply:
what the o_O ?? I have no clue why there are here, will remove them of course